src/main/contextMenu.ts

```diff
@@ -51,11 +51,7 @@ export default class ContextMenu {
     reload = () => {
         this.dispose();
 
-        /**
-         * Work-around issue with passing `WebContents` to `electron-context-menu` in Electron 11
-         * @see https://github.com/sindresorhus/electron-context-menu/issues/123
-         */
-        const options = {window: {webContents: this.view.webContents, inspectElement: this.view.webContents.inspectElement.bind(this.view.webContents), isDestroyed: this.view.webContents.isDestroyed.bind(this.view.webContents), off: this.view.webContents.off.bind(this.view.webContents)} as unknown as WebContents, ...this.menuOptions};
+        const options = {window: this.view, ...this.menuOptions};
```

Comment:
This workaround is no longer needed, it was fixed with the patch I sent to `electron-context-menu` a few versions back, I just never noticed until now XD